scheduler/reconciler/reconcile_cluster.go

```diff
@@ -490,8 +490,6 @@ func (a *AllocReconciler) computeGroup(group string, all allocSet) (*ReconcileRe
 		migrate = migrate.difference(stopAllocSet)
 		lost = lost.difference(stopAllocSet)
 		disconnecting = disconnecting.difference(stopAllocSet)
-		reconnecting = reconnecting.difference(stopAllocSet)
-		ignore = ignore.difference(stopAllocSet)
```

Comment:
My only concern is that this leaves reconnecting and ignore in a potentially surprising state in a huge func. If someone adds some code in the future 100+ lines down this func, they might assume these variables are in an updated state when they aren't any longer.

Ideally this func would be smaller so that we could drop this code and still have all variables consistently updated throughout the func.

Sounds like a problem for another day though, so feel free to ship this small improvement. 😁

Reply:
We could add a note in the comment above here:

```
		// Stop the reconciled allocations and remove them from the other sets
		// since they have been already handled.
```